src/test/java/seedu/exercise/storage/StorageManagerTest.java

```diff
@@ -25,8 +25,9 @@
     @BeforeEach
     public void setUp() {
         JsonExerciseBookStorage exerciseBookStorage = new JsonExerciseBookStorage(getTempFilePath("ab"));
+        JsonRegimeBookStorage regimeBookStorage = new JsonRegimeBookStorage(getTempFilePath("cd"));
         JsonUserPrefsStorage userPrefsStorage = new JsonUserPrefsStorage(getTempFilePath("prefs"));
```

Comment:
"ef"

Nothing important here. But I think the first TempFilePath for exerciseBookStorage "ab" might be a short form for 'address-book'.

Reply:
ok